policyeval/base_worker.go

```diff
@@ -323,6 +325,10 @@ func (h *checkHandler) start(ctx context.Context, currentStatus *sdk.TargetStatu
 		return &sdk.ScalingAction{Direction: sdk.ScaleDirectionNone}, nil
 	}
 
+	for _, m := range h.checkEval.Metrics {
+		h.logger.Trace("metric result", "ts", m.Timestamp, "value", m.Value)
+	}
```

Comment:
I wonder if this is worth wrapping with `if h.logger.IsTrace() {}` so that we only enter the loop if we are going to print the metrics and this avoids looping tens/hundreds of times when we don't need to.